### ISSUE TITLE
NTR: Dodecanedioic acid high in urine (Lab: urine tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8861]
New term request Dodecanedioic acid high in urine (elthat_130323220911): C12 dicarboxylic acid

**Comment by pnrobinson** [comment]
@MickeySegal
I cannot find this. What is the context?

**Comment by MickeySegal** [comment]
https://www.ncbi.nlm.nih.gov/books/NBK1527/
"Urine organic acids that demonstrate elevated dodecanedioic acid during acute crisis and for several days following"

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000912
- label: Elevatged urine dodecanedioic acid level
- parents: [Organic aciduria (HP:0001992)](https://hpo.jax.org/app/browse/term/HP:0001992)
- pmid: 
- def: The amount of dodecanedioic acid in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8861
- synonyms:
